pippy/ManualPipelineStage.py

```diff
@@ -220,36 +223,63 @@ def stage_global_rank(peer_rank):
             (self.group_rank + 1) % self.group_size
         )
 
+        # Receive info during forward
+        # TODO: create args_recv_info lazily? (same needed for PipelineStage)
+        for chunk_id in range(self.chunks):
+            self.set_requires_grad[chunk_id] = False
+            if not self.is_first:
+                self.args_recv_info[chunk_id] = tuple(
+                    [
+                        RecvInfo(
+                            f"recv_for_{self.stage_index}_from_{self.stage_index - 1}",
+                            self.stage_index - 1,
```

Comment:
nit: Can you added a comment for this line stating the assumption of receiving from -1 stage?

Reply:
yep, good point. thanks!